### PR TITLE
Fix samples

### PR DESCRIPTION
**Pull request by kichalla**
@Tratcher @sebastienros

### CODE REVIEW

File: samples/MemoryCacheConcurencySample/project.json [hunk 1]
```diff
@@ -1,20 +1,22 @@
 {
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+
   "dependencies": {
     "Microsoft.Extensions.Caching.Memory": "1.0.0-*"
   },
-  "commands": {
-    "MemoryCacheConcurencySample": "MemoryCacheConcurencySample"
```
Comment (Tratcher) [inline samples/MemoryCacheConcurencySample/project.json]: Do we have tooling yet that can launch the samples without a command? If not, leave the commands for now.

**Comment by Tratcher** [comment]
:shipit:

File: samples/MemoryCacheConcurencySample/project.json (hunk 1 above)
Comment (kichalla) [inline samples/MemoryCacheConcurencySample/project.json]: With the latest tooling that I have I see that it works fine without a command.

**Comment by kichalla** [comment]
f80c13ee9dbcf53a5c4c802ddfe40eb0702fb9df

**Pull request closed without merging**